backend/market_data/services/tick_archive_provider.py

```
@@ -203,15 +204,21 @@ def fetch_twse_public_snapshot(
+            except requests.RequestException as exc:
+                error_type = type(exc).__name__
+                logger.error(
+                    "Failed to fetch TWSE public snapshot after CA download "
+                    "symbol_count=%s error_type=%s",
                     len(symbols),
+                    error_type,
                 )
-            except Exception:
-                logger.exception(
-                    "Failed to download TWSE MIS CA bundle symbol_count=%s",
+            except Exception as exc:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)